src/main/java/co/rsk/federate/FedNodeRunner.java

```diff
@@ -162,10 +163,18 @@ private void startBookkeepingServices() throws SignerException, HSMClientExcepti
         HSMClientProtocol protocol =
             hsmClientProtocolFactory.buildHSMClientProtocolFromConfig(powHsmConfig);
 
-        int hsmVersion = protocol.getVersion();
+        int version = protocol.getVersion();
+        HSMVersion hsmVersion;
+        try {
+            hsmVersion = HSMVersion.valueOf("V" + version);
+        } catch (IllegalArgumentException e) {
+            logger.debug("[run] Unsupported HSM version  {}", version);
+            throw new HSMUnsupportedVersionException("Unsupported HSM version " + version);
+        }
+
```

Comment:
```suggestion        
        if (!HSMVersion.isPowHSM(version)) {
             throw new HSMUnsupportedVersionException("Unsupported HSM version " + version);
        }

        logger.debug("[run] Using HSM version {}", version);

        hsmBookkeepingClient = buildBookKeepingClient(protocol, powHsmConfig);
        hsmBookkeepingService = buildBookKeepingService(hsmBookkeepingClient, powHsmConfig);
```

i think that should do it right? and we'll be also applying early return approach

Reply:
why? it wasn't considering it before when starting bookkeeping client right?

Reply:
oh i see, we don't wanna throw an error when hsm version 1